tools/tpm2_clearcontrol.c

```diff
@@ -11,30 +12,29 @@
 
 typedef struct clearcontrol_ctx clearcontrol_ctx;
 struct clearcontrol_ctx {
-    bool clear;
-    bool platform;
-
+    TPMI_YES_NO disable_clear;
+    ESYS_TR rh;
     struct {
         char *auth_str;
         tpm2_session *session;
     } auth;
 };
 
-static clearcontrol_ctx ctx;
+static clearcontrol_ctx ctx = {
+    .rh = ESYS_TR_RH_PLATFORM,
+    .disable_clear = 0,
+};
 
 static bool clearcontrol(ESYS_CONTEXT *ectx) {
 
-    ESYS_TR rh = ctx.platform ? ESYS_TR_RH_PLATFORM : ESYS_TR_RH_LOCKOUT;
-    TPMI_YES_NO disable = ctx.clear ? 0 : 1;
-
-    LOG_INFO ("Sending TPM2_ClearControl(%s) command on %s",
-            ctx.clear ? "CLEAR" : "SET",
-            ctx.platform ? "TPM2_RH_PLATFORM" : "TPM2_RH_LOCKOUT");
+    LOG_WARN ("Sending TPM2_ClearControl(%s) disableClear command with auth handle %s",
```

Comment:
LOG_INFO, why is this warning now?

Reply:
@williamcroberts, for what the tool does - Disable owner capability to clear the TPM - I thought that it should at least warn the user of this choice. So future failed attempts to clear the TPM with owner auth will have a context.

Reply:
Your warning doesn't convey this. Your warning just says, "hey ive run this command with these arguments". If your trying to convey spec behavior, then say that LOG_WARN("Since your clearing the TPM future clears may fail");

I have no idea exactly what your trying to say here. I would just discard it, but if their is some higher level important information to convey, then do that. Think of a good code comment, don't tell me what the code does (I can read that or im executing the command I know that), tell me why it matters.

Reply:
Reverted it to LOG_INFO